File: 백준/Bronze/4153.직각삼각형/직각삼각형.cc
#include <stdio.h>

int x[100], y[100], z[100], i;

main() {
    
    while(1) {
        scanf("%d %d %d", &x[i], &y[i], &z[i]);
        
        if (x[i] == 0) break;
        i++;
        
    }
    
    for (int j=0; j<i; j++) {
        if (
			x[j]*x[j] + y[j]*y[j] == z[j]*z[j]
			|| y[j]*y[j] + z[j]*z[j] == x[j]*x[j]
			|| x[j]*x[j] + z[j]*z[j] == y[j]*y[j]) printf("right\n");
        else printf("wrong\n");
    }
    
}

File: 백준/Bronze/4153.직각삼각형/직각삼각형.py
while True :
  a, b, c = map(int, input().split())
  if (a and b and c) == 0 :
    break
  lst = [a, b, c]
  lst.sort()
  if lst[0]**2 + lst[1]**2 == lst[2]**2 :
      print('right')
  else :
      print('wrong')